### PR TITLE
[WIP] Add Documentation to install Liqo on AKS

### PR DESCRIPTION
**Pull request by aleoli**
# Description

This pr adds the documentation on how to install Liqo on an AKS managed cluster

Fixes #(issue)

### CODE REVIEW

File: docs/pages/User/Installation/aks.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+## Deploy Liqo
+
+### One Line installer
+
+#### Export Variables
+
+The Liqo one line installer needs some environment variables to know where the Liqo components are accessible from the
+external world and how the local Kubernetes installation is configured.
+
+In particular we have to export the following environment variables:
```
Comment (palexster) [inline docs/pages/User/Installation/aks.md:80]: I really liked how the table is rendered in the very next section. Can we use a table also for the parameters listed here? What do you think?

Comment (aleoli) [reply docs/pages/User/Installation/aks.md:80]: Yes sure, I can add it

File: docs/pages/User/Installation/aks.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+```bash
+curl --insecure https://auth.f83c28d9ce1449b2bb45.westeurope.aksapp.io/ids
+```
+
+```json
+{"clusterId":"4e33e2dc-b2e3-4052-9a74-d18f6d8cdab2","clusterName":"LiqoCluster5852","guestNamespace":"liqo"}
+```
+
+Congratulations! Liqo is now up and running on your AKS cluster, you can now peer with other Liqo instances!
```
Comment (palexster) [inline docs/pages/User/Installation/aks.md:161]: I would add a few commands explaining how we can use the aforementioned data to enable a peering on the other cluster.

**Pull request closed without merging**